dlp/system-test/inspect.test.js

```diff
@@ -309,4 +309,38 @@ describe('inspect', () => {
     assert.notMatch(outputB, /EMAIL_ADDRESS/);
     assert.match(outputB, /PHONE_NUMBER/);
   });
+
+  // dlp_inspect_custom_regex
+  it('should inspect a string using custom regex', () => {
+    const string = 'Patients MRN 444-5-22222';
+    const custRegex = '[1-9]{3}-[1-9]{1}-[1-9]{5}';
+    const output = execSync(
+      `node inspectWithCustomRegex.js ${projectId} "${string}" UNLIKELY 0 "" ${custRegex} true`
```

Comment:
Quote the regex similar to string to avoid user confusion and test failures if the regex is something that bash could expand. (e.g. `[ab][cd]` would become `ac` if such a file existed)